pkg/capabilities/errors.go

```diff
@@ -0,0 +1,44 @@
+package capabilities
+
+import (
+	"fmt"
+	"strings"
+
+	"kernel.org/pub/linux/libs/security/libcap/cap"
+)
+
+type MissingCapabilitiesError struct {
+	missingCaps []cap.Value
+}
+
+type DropCapabilitiesError struct {
+	Err error
+}
+
+func newErrMissingCapabilities(missingCaps []cap.Value) *MissingCapabilitiesError {
```

Comment:
Since there's no special logic to creating the error I think the new functions are overkill, just export the struct and fields. No need for encapsulation here, especially since these should be returned as `error`s anyway so the interface will encapsulate naturally.